### PR TITLE
179: format dates in dashboard table/cards as UTC

### LINKED ISSUE #179: Inventory dashboard is showing dates in local timezone instead of UTC

### PR DESCRIPTION
**Pull request by rpeterman-gp**
Closes #179 

### Changes
- Add `'UTC'` timezone param to `data` filter used to display dates in flights table.
- Updates `campaignFlightDates` pipe to format dates with `utc` moment function.

### CODE REVIEW

File: src/app/dashboard/campaign-card/campaign-flight-dates.pipe.ts [hunk 1]
```diff
@@ -27,8 +28,9 @@ export class CampaignFlightDatesPipe implements PipeTransform {
 
   transform(flights: Flight[]): string {
     const dates = this.flightDates(flights);
+    const dateFormat = 'M/D';
```
Comment (sandikbarr) [inline src/app/dashboard/campaign-card/campaign-flight-dates.pipe.ts:31]: The spec was expecting these dates to not be zero padded (from original design.)

File: src/app/dashboard/flight-table/flight-table.component.html [hunk 2]
```diff
@@ -57,15 +57,15 @@
       <th mat-header-cell *matHeaderCellDef mat-sort-header>
```
Comment (sandikbarr) [inline src/app/dashboard/flight-table/flight-table.component.html:57]: Ran a prettier format on save here to keep prettier's opinionated formatting in this file.

**Review by sandikbarr — APPROVED** [review]
@rpeterman-gp made a couple small changes, ready to be merged after you look it over

**Comment by rpeterman-gp** [comment]
@sandikbarr Your changes look good to me. Thanks!

**Pull request merged**